src/compiler/emitter.ts

```diff
@@ -3,6 +3,190 @@
 
 /* @internal */
 namespace ts {
+    export const defaultInitCompilerOptions: CompilerOptions = {
+        module: ModuleKind.CommonJS,
+        target: ScriptTarget.ES3,
+        noImplicitAny: true,
+        outDir: "built",
+        rootDir: ".",
+        sourceMap: false,
+    }
+
+    export function buildConfigFile(writer: EmitTextWriter, compilerOptions: CompilerOptions, fileNames?: string[], excludes?: string[]) {

```

Comment:
I do not think this belongs in the emitter. i think this should be commandlineParser.ts